test/test_component_fluent/test_component_fluent.idl

```diff
@@ -0,0 +1,21 @@
+import "Windows.Foundation.idl";
```

Comment:
Since this is always-on, you can just add this to the existing `test_component` project and test it from the existing `test` project.